### ISSUE TITLE
Documentation is incorrect for CommonJS

### ISSUE THREAD

**Issue by jrop** [lukeed/fromNow#2]
The documentation says that one can do:
```js
const fromNow = require('fromnow')
fromNow('12/31/2010')
// => '5 years ...'
```

However, in CommonJS environments, the module exports the `factory` not the `fromNow` function, so the only way to get this to work is to do:
```js
const fromNow = require('fromnow')() // <= invoke factory function
fromNow('12/31/2010')
// => '5 years ...'
```

**Comment by jrop** [comment]
It could be that your UMD implementation is off; according to [this template](https://github.com/umdjs/umd/blob/master/templates/commonjsStrictGlobal.js), instead of:

```js
module.exports = factory
```

It should be

```js
factory(exports)
```

**Comment by lukeed** [comment]
Hey, thanks for this.

Admittedly I've not checked in on this repo in a while.

Can you do me a favor & see if `module.exports = factory()` works?

**Comment by jrop** [comment]
I have verified that your modification works

**Comment by lukeed** [comment]
Thanks! I'll push a quick update

**Comment by lukeed** [comment]
Closed in 9339d82f0c55d5a7efa04608e242eba0f9f5ad1a
